_unittest_ironpython/run_unittests.py

```diff
@@ -10,8 +11,12 @@
 
 os.environ["UNITTEST_CURRENT_TEST"] = "1"
 run_dir = os.path.abspath(os.path.dirname(__file__))
-print(run_dir)
 
+args_env = os.environ.get("RUN_UNITTESTS_ARGS", "")
```

Comment:
@koubaa We might not need to create a variable here if it is only used one time in the code below. You can use it directly line 18.